mltrace/cli/cli.py

```diff
@@ -223,6 +227,22 @@ def recent(limit: int, address: str = ""):
         show_info_card(id)
 
 
+@mltrace.command("inspect")
+@click.option("--address", help="Database server address")
+@click.argument("component_run_id")
+def inspect(component_run_id, address: str = ""):
+    """
+    CLI to inspect a specific component run id.
+    """
+    # Set address
+    if address and len(address) > 0:
+        set_address(address)
+    # Get the recent ids
+    component_run_ids = get_recent_run_ids()
```

Comment:
Your logic here only shows info for the component run ID if it was in the most recent run IDs. We want to be able to show info regardless. You don't need the if statement -- you can just directly call `show_info_card`

Reply:
done.